404.css

```diff
@@ -0,0 +1,9 @@
+footer {
+  color: #d6351c;
+  font-family: monospace;
```

Comment:
wouldn't this have no effect on the text since we already wrapped it with `<code>` tags?